app/src/main/java/ru/netology/fmhandroid/adapter/PatientListAdapter.kt

```diff
@@ -35,7 +36,18 @@ class PatientListAdapter(
         fun bind(patient: Patient) {
             binding.apply {
```

Comment:
Исключительно стилистическое предложение. Так код неменого влево уйдёт
```suggestion
        fun bind(patient: Patient) = with (binding) {
```

Reply:
Отлично! Исправлено.